twisted/mail/alias.py

```diff
@@ -742,7 +742,8 @@ def __init__(self, items, *args):
                 except:
                     log.err("Invalid filename in alias file %r" % (addr[1:],))
                 else:
-                    addr = ' '.join([l.strip() for l in f])
+                    with f:
```

Comment:
minor comment: maybe we can get rid of the else branch and just use `with open(addr[1:]) as f:` ... I don't see why we need the else branch here.